### PR TITLE
Add a chefworkstation docker container

### PR DESCRIPTION
**Pull request by tas50**
We have one for dk. Even though on the CLI it's the same we should include that.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by tas50** [comment]
Yeah this is really just for marketing sake. We can move everything off the DK containers

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:18.04
+MAINTAINER Chef Software, Inc. <docker@chef.io>
+
+ARG CHANNEL=stable
+ARG VERSION=0.2.53
+ENV DEBIAN_FRONTEND=noninteractive \
+    PATH=/opt/chefworkstation/bin:/opt/chefworkstation/embedded/bin:/root/.chefdk/gem/ruby/2.6.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
Comment (tyler-ball) [inline Dockerfile]: We install to `/opt/chef-workstation`. I'll update this and rebase on master.

File: .expeditor/config.yml [hunk 2]
```diff
@@ -8,19 +8,18 @@ product_key:
 slack:
   notify_channel: chef-ws-notify
 
+docker_images:
+  - chefworkstation
```
Comment (tyler-ball) [inline .expeditor/config.yml]: What is the reason for this over `chef-workstation`?

Comment (tas50) [reply .expeditor/config.yml]: it could go either way. Now is our chance

Comment (tyler-ball) [reply .expeditor/config.yml]: I vote `chef-workstation` since it is easier to read

Comment (tas50) [reply .expeditor/config.yml]: Push away

**Comment by tas50** [comment]
@tduffield what am I missing in the expeditor config here?

File: .expeditor/config.yml [hunk 3]
```diff
@@ -49,13 +49,21 @@ pipelines:
...
+    actions:
+      - built_in:build_docker_image
+  - workload: artifact_published:current:chef-workstation:{{version_constraint}}
+    actions:
+      - built_in:tag_docker_image
   - workload: artifact_published:stable:chef-workstation:{{version_constraint}}
     actions:
       - built_in:rollover_changelog
+      - bash:.expeditor/update_dockerfile.sh
+      - built_in:tag_docker_image
```
Comment (tduffield) [inline .expeditor/config.yml]: Your actions here are out of order. `tag_docker_image` is a post-commit action, while bash actions are (by default) pre-commit actions. To address the expeditor config issue you'll want to move this action to the end of the list (or add a `post_commit: true`) to the subsequent bash actions.

**Pull request merged**